### ISSUE TITLE
io.fabric8:docker-maven-plugin:0.27 regression

### ISSUE THREAD

**Issue by jmayday** [fabric8io/docker-maven-plugin#1093]
### Description
Failed to execute goal io.fabric8:docker-maven-plugin:0.27.0:start

I suspect that it's related to this change: 
_Fix for creating volumes with proper configuration during "docker:start" (#986)_

and change itself:
https://codecov.io/gh/fabric8io/docker-maven-plugin/pull/1007/diff?src=pr&el=tree#diff-c3JjL21haW4vamF2YS9pby9mYWJyaWM4L21hdmVuL2RvY2tlci9TdGFydE1vam8uamF2YQ==

### Info
maven setup:
```
<plugin>
        <groupId>io.fabric8</groupId>
        <artifactId>docker-maven-plugin</artifactId>
        <version>0.27.0</version>
        <configuration>
          <registry>
            docker.io
          </registry>
          <images>
            <image>
              <name>mysql:5.6.41</name>
              <alias>mysql</alias>
              <run>
                <env>
                  <MYSQL_ROOT_PASSWORD>xyz</MYSQL_ROOT_PASSWORD>
                  <MYSQL_DATABASE>xyz</MYSQL_DATABASE>
                  <MYSQL_USER>xyz</MYSQL_USER>
                  <MYSQL_PASSWORD>xyz</MYSQL_PASSWORD>
                </env>
                <ports>
                  <port>${mysql.test.db_port}:3306</port>
                </ports>
                <wait>
                  <time>8000</time>
                </wait>
              </run>
            </image>
          </images>
        </configuration>
        <executions>
          <execution>
            <id>start</id>
            <phase>pre-integration-test</phase>
            <goals>
              <goal>start</goal>
            </goals>
          </execution>
          <execution>
            <id>stop</id>
            <phase>post-integration-test</phase>
            <goals>
              <goal>stop</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
```

Jenkins stacktrace:

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.27.0:start (start) on project xyz: Execution start of goal io.fabric8:docker-maven-plugin:0.27.0:start failed. NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.27.0:start (start) on project xyz: Execution start of goal io.fabric8:docker-maven-plugin:0.27.0:start failed.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution start of goal io.fabric8:docker-maven-plugin:0.27.0:start failed.
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: java.lang.NullPointerException
	at io.fabric8.maven.docker.StartMojo.prepareStart(StartMojo.java:349)
	at io.fabric8.maven.docker.StartMojo.executeInternal(StartMojo.java:149)
	at io.fabric8.maven.docker.AbstractDockerMojo.execute(AbstractDockerMojo.java:225)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

```

* d-m-p version :
* Maven version (`mvn -v`) :
```
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T20:33:14+02:00)
Maven home: C:\Program Files\apache-maven-3.5.4\bin\..
Java version: 9.0.4, vendor: Oracle Corporation, runtime: C:\Program Files\openjdk-9.0.4_windows-x64_bin\jdk-9.0.4
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```
* Docker version :
```
Version 18.06.1-ce-win73 (19507)
Channel: stable
b0f14f1
```

**Comment by rhuss** [comment]
Fixed with #1092 . Going to release 0.27.1 this evening ...

**Comment by rhuss** [comment]
right, this is a *different* regression. Sorry for that, should have checked bind configuration without volumes. 

I'm doing a fix asap ...

**Comment by jmayday** [comment]
I closed issue as 0.27.1 fixes the issue
